Dockerfile

```diff
@@ -84,3 +85,26 @@ RUN <<EOT
         Pkg.build();
     '
 EOT
+
+# Add the RetrivalToolbox package
+RUN julia -e 'using Pkg; Pkg.add(url="https://github.com/US-GHG-Center/RetrievalToolbox.jl");'
+
+# Add the demo notebook
+RUN git clone https://github.com/US-GHG-Center/IWGGMS21-Demo ~/IWGGMS21-Demo
+
+# Run the precompile/build within Jupyter (somehow that is necessary)
+COPY precompile_build.ipynb ./
+RUN jupyter execute precompile_build.ipynb && rm precompile_build.ipynb
+
+# Add the XRTM path as an env
+ENV XRTM_PATH=/opt/xrtm
+
+# Download and install JuliaMono font for good rendering of plotting text
+RUN <<EOT
+   curl -OL https://github.com/cormullion/juliamono/releases/download/v0.060/JuliaMono-ttf.tar.gz
+   mkdir ~/.fonts
+   cd ~/.fonts
+   tar -xvzf ~/JuliaMono-ttf.tar.gz
+   rm ~/JuliaMono-ttf.tar.gz
+EOT
```

Comment:
Again, we cannot place anything in the home dir. I suggest determining where else you can install fonts in the system so that they can be discovered. This may also require moving this block _before_ the line `USER ${NB_USER}`, as I suspect installing fonts in some recognized system location may require root permissions.